ci/parse_ci_envs.sh

```diff
@@ -32,6 +32,10 @@ if [ -n "${STAGE_NAME:?}" ]; then
       : "${TARGET:=ubuntu22}"
       : "${REPO_SPEC:=ubuntu-22.04}"
       ;;
+    *Ubuntu\ 22.04*|*ubuntu2204*)
```

Comment:
(lint) This pattern never matches because of a previous pattern on line 30. [SC2222]
(lint) This pattern never matches because of a previous pattern on line 30. [SC2222]